### ISSUE TITLE
Update comparison https://github.com/tpope/vim-repeat

### ISSUE THREAD

**Issue by Konfekt** [kana/vim-repeat#1]
Dear Kana,

I stumbled upon your plugin because my ctrl-r key is overwritten and vim-repeat does not respect it. Could you give an updated account of the issues you found with vim-repeat? Do you consider your plugin stable?

**Comment by tyru** [comment]
You can see comparison by `:help repeat-vs-original`

**Comment by Konfekt** [comment]
Yes, the question was if all these points still apply because repeat has been updated in the meanwhile.
